src/main/java/org/example/ElectronicBook.java

```diff
@@ -0,0 +1,45 @@
+package org.example;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString(callSuper = true)
+public class ElectronicBook extends book {
+    private String format;
+    private long fileSizeInBytes;
+    private static final int BYTES_PER_PAGE_COUNT = 2048;
+    private int year;
+    private String author;
+
+    public ElectronicBook(String title, String author, String publisher, int year, String genre, String format, long fileSizeInBytes) {
+        super(title, author, publisher, year, genre);
+        setTitle(title);
+        setAuthor(author);
+        setPublisher(publisher);
+        setYear(year);
+        setGenre(genre);

```

Comment:
those calls are redundant - you've already passed title, author, publisher, year, genre to the parent constructor